Tests/web3swiftTests/localTests/ABIEncoderTest.swift

```diff
@@ -202,22 +202,21 @@ class ABIEncoderTest: XCTestCase {
         XCTAssertEqual(hexData, "000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000047465737400000000000000000000000000000000000000000000000000000000")
         hexData = ABIEncoder.encode(types: [.array(type: .uint(bits: 8), length: 0)], values: [[1, 2, 3, 4]])?.toHexString()
         XCTAssertEqual(hexData?[0..<64], "0000000000000000000000000000000000000000000000000000000000000020")
-        XCTAssertEqual(hexData, "000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000004")
-
+        XCTAssertEqual(hexData, "0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000401020304")
         // This one shouldn't have data offset
         hexData = ABIEncoder.encode(types: [.array(type: .uint(bits: 8), length: 4)], values: [[1, 2, 3, 4]])?.toHexString()
         // First 32 bytes are the first value from the array
-        XCTAssertEqual(hexData?[0..<64], "0000000000000000000000000000000000000000000000000000000000000001")
-        XCTAssertEqual(hexData, "0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000004")
+//        XCTAssertEqual(hexData?[0..<32], "0000000000000000000000000000000000000000000000000000000000000001")
```

Comment:
Wasn't sure about this line. @JeneaVranceanu @yaroslavyaroslav 

Is it actually needed or could it be deleted? Because of the bug fix of uint bits this hex string is now signficantly shorter.